easybuild/easyconfigs/s/Salmon/Salmon-1.0.0-gompi-2019b.eb

```diff
@@ -0,0 +1,53 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage:     https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::     Robert Qiao <rob.qiao@flinders.edu.au>
+# License::     GLPv2
+#
+# Notes::
+##
+
+easyblock = 'CMakeMake'
+
+name = 'Salmon'
+version = '1.0.0'
+
+github_account = 'COMBINE-lab'
+homepage = 'https://github.com/%(github_account)s/%(namelower)s'
+description = """Salmon is a wicked-fast program to produce a highly-accurate,
+ transcript-level quantification estimates from RNA-seq data."""
+
+toolchain = {'name': 'gompi', 'version': '2019b'}
+
+toolchainopts = {'pic': True, 'openmp': True}
+
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['fd8443cd6ac0c8dd565d1341bea955753933336390c7caaead37a967d4aaa739']
+
+builddependencies = [
+    ('CMake', '3.15.3', '', ('GCCcore', '8.3.0')),
+    ('pkg-config', '0.29.2', '', ('GCCcore', '8.3.0')),
+    ('jemalloc', '5.2.1', '', ('GCCcore', '8.3.0')),
+]
+
+dependencies = [
+    ('Boost', '1.70.0'),
```

Comment:
@robqiao this should be version 1.71.0 (unless that version does not work, in that case you have to create an easyconfig for version 1.70 with the 2019b toolchain).